apps/dashboard/src/components/template-store/templates/authentication/organization-invitation.ts

```
@@ -1,5 +1,5 @@
-import { StepTypeEnum, WorkflowCreationSourceEnum } from '@novu/shared';
 import { WorkflowTemplate } from '../types';
+import { StepTypeEnum, WorkflowCreationSourceEnum } from '@novu/api/models/components';
```

ESLint

[error] `@novu/api/models/components` import should occur before import of `../types`

(import/order)